packages/tame-global-error-object/test/tameGlobalErrorObject.test.js

```diff
@@ -4,18 +4,44 @@ import tameGlobalErrorObject from '../src/main.js';
 
 const { test } = tap;
 
-test('tameGlobalErrorObject', t => {
-  t.plan(3);
+test('tameGlobalErrorObject - captureStackTrace', t => {
+  t.plan(7);
 
   const restore = captureGlobals('Error');
 
   tameGlobalErrorObject();
```

Comment:
We never did a `lockdown()` and so is outside of SES. I suppose it works anyway, and is a fine thing to test. But is that what you wanted to test?

Is this intended to be in test262 style? (I'm guessing. I don't know what the current style looks like.)